ExtIO_sddc/ExtIO_sddc.cpp

```diff
@@ -112,7 +112,19 @@ bool __declspec(dllexport) __stdcall InitHW(char *name, char *model, int& type)
 	if (!gbInitHW)
 	{
 		// do initialization
-		h_dialog = CreateDialog(hInst, MAKEINTRESOURCE(IDD_DLG_MAIN), NULL, (DLGPROC)&DlgMainFn);
+		// verify if HDSDR host name
+		SDR_progname[0] = 0;
```

Comment:
this is not needed as you declare this as BSS already.

Reply:
Thanks